ui/src/components/Form/SelectBox/index.tsx

```diff
@@ -22,27 +22,28 @@ interface FileFieldProps {
   options: SelectFieldItem[];
   isLoading?: boolean;
   onSelect?: any;
```

Comment:
We don't need to make this type any
```suggestion
  onSelect: (item: SelectFieldItem) => void;
```